src/rpc/rawtransaction.cpp

```diff
@@ -60,6 +59,17 @@ void ScriptPubKeyToJSON(const CScript& scriptPubKey, UniValue& out, bool fInclud
     out.push_back(Pair("addresses", a));
 }
 
+namespace {
+    void fillStdFields(UniValue & out, CTxIn const & txin) {
+        out.push_back(Pair("txid", txin.prevout.hash.GetHex()));
+        out.push_back(Pair("vout", (int64_t)txin.prevout.n));
```

Comment:
My idea is we replace `txid` and `vout` fields to something like `anonymityGroup` will be more useful since `txid` will be always zeroes for Sigma spend. @a-bezrukov @martun what do you guys think?

Reply:
I think it's a good idea. Please tell me which fields we need. Easy available there are CoinSpendV3 fields and groupId.

Reply:
Anonymity group for Sigma is in `txin.prevout.n`.

Reply:
Done